corona-warn-app/cwa-app-android
# EOL obsolete card (EXPOSUREAPP-6700)

Removed obsolete Home Screen "Risk Calculation stopped" (Risiko-Überprüfung beendet) card and corresponding functions and events.

How to test:
1) Submit positive test (PCR or Rapid)
2) Check home screen - there should be no more "Risk Calculation stopped" card (below statistics cards).
